bigframes/core/global_session.py

```diff
@@ -36,7 +41,15 @@ def close_session() -> None:
 
     with _global_session_lock:
         if _global_session is not None:
-            _global_session.close()
+            try:
+                _global_session.close()
+            except google.auth.exceptions.RefreshError as e:
+                session_id = _global_session.session_id
+                warnings.warn(
+                    "Session cleanup failed for session with"
+                    + "id: {session_id}".format(session_id=session_id)
```

Comment:
Since the session is scoped to the project and location, could we include these in the output, too?

Reply:
Added